src/frontend/js/components/SearchFilterGroupModal/index.tsx

```diff
@@ -119,51 +151,61 @@ export const SearchFilterGroupModal = ({ filter }: SearchFilterGroupModalProps)
-                        type: CourseSearchParamsAction.filterAdd,
-                      });
-                      setModalIsOpen(false);
-                    }}
-                  >
-                    {value.human_name}&nbsp;
-                    {value.count || value.count === 0 ? `(${value.count})` : ''}
-                  </button>
-                </li>
-              ))}
-            </ul>
-          )}
-        </fieldset>
-        <button className="search-filter-group-modal__close" onClick={() => setModalIsOpen(false)}>
-          <FormattedMessage {...messages.closeButton} />
-        </button>
+        <div className="search-filter-group-modal__inner-container">
+          <fieldset className="search-filter-group-modal__form">
+            <legend className="search-filter-group-modal__form__title">
+              <FormattedMessage
+                {...messages.modalTitle}
+                values={{ filterName: filter.human_name }}
+              />
+            </legend>
+            <input
+              aria-label={intl.formatMessage(messages.inputLabel)}
+              className="search-filter-group-modal__form__input"
+              onChange={(event) => {
+                setQuery(event.target.value);
+              }}
+              placeholder={intl.formatMessage(messages.inputPlaceholder, {
+                filterName: filter.human_name,
+              })}
+            />
+            {error ? (
+              <div className="search-filter-group-modal__form__error">
+                <FormattedMessage {...messages.error} values={{ filterName: filter.human_name }} />
+              </div>
+            ) : query.length > 0 && query.length < 3 ? (
+              <div className="search-filter-group-modal__form__error">
+                <FormattedMessage {...messages.queryTooShort} />
+              </div>
+            ) : (
+              <ul className="search-filter-group-modal__form__values">
+                {values.map((value) => (
+                  <li className="search-filter-group-modal__form__values__item" key={value.key}>
+                    <button
+                      onClick={() => {
+                        dispatchCourseSearchParamsUpdate({
+                          filter,
+                          payload: value.key,
+                          type: CourseSearchParamsAction.filterAdd,
+                        });
+                        setModalIsOpen(false);
+                      }}
+                    >
+                      {value.human_name}&nbsp;
+                      {value.count || value.count === 0 ? `(${value.count})` : ''}
+                    </button>
+                  </li>
+                ))}
+              </ul>
+            )}
+          </fieldset>
+          <button
+            className="search-filter-group-modal__close"
+            onClick={() => setModalIsOpen(false)}
+          >
+            <FormattedMessage {...messages.closeButton} />
+          </button>
```

Comment:
The label is no more centered. 🤔

Reply:
It's centered for me on Latest firefox & Safari 13. In which browser are you seeing this?

Reply:
Firefox 87.0 on Mac OS.